docs/api/puppeteer.elementhandle.contentframe.md

```diff
@@ -44,3 +44,17 @@ this
 **Returns:**
 
 Promise&lt;[Frame](./puppeteer.frame.md)&gt;
+
+## ElementHandle.contentFrame() (Overload 1) method
```

Comment:
@Lightning00Blade I think we could try to produce something similar to https://typedoc.org/example/functions/overloadedFunction.html#overloadedFunction I.e. we should put the parameter name and return type into the title of the of the overload. Any thoughts on how to achieve this?

Reply:
Probably we can use ApiParameterListMixin.parameters and ApiReturnTypeMixin to generate a comma-separated list of params (without type names) + return type.